website/docs/d/aaep_to_domain.html.markdown

```diff
@@ -0,0 +1,44 @@
+---
+subcategory: "Access Policies"
+layout: "aci"
+page_title: "ACI: aci_aaep_to_domain"
+sidebar_current: "docs-aci-data-aaep-to-domain"
+description: |-
+  Data source for ACI Attachable Access Entity Profile - VMM, Physical or External domain interfaces.
+---
+
+# aci_aaep_to_domain #
+
+Data source for ACI Attachable Access Entity Profile - VMM, Physical or External domain interfaces.
+
+
+## API Information ##
+
+* `Class` - infraRsDomP
+* `Distinguished Name` - uni/infra/attentp-{name}/rsdomP-[{tDn}]
+
+## GUI Information ##
+
+* `Location` - Fabric -> Access Policies -> Policies -> Global -> AAEP
+
+
+
+## Example Usage ##
+
+```hcl
+data "aci_aaep_to_domain" "foo_aaep_to_domain" {
+  attachable_access_entity_profile_dn = aci_attachable_access_entity_profile.fooattachable_access_entity_profile.id
+  domain_dn                           = aci_l3_domain_profile.fool3_domain_profile.id
+}
+```
+
+## Argument Reference ##
+
+* `attachable_access_entity_profile_dn` - (Required) Distinguished name of parent AttachableAccessEntityProfile object.
+* `annotation` - (Optional) Annotation of the object Domain.
```

Comment:
not a necessary argument so can be removed from here and is documented in the attribute section.